RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -454,8 +455,10 @@ class ProjectLoader {
             logger.postRareInfo("Loading information configuration context for ${projectID} from ${iccProject.file}")
         }
 
-        //Validate the project icc
-        XSDValidator.validateTree(iccProject);
+        if (!XSDValidator.validateTree(iccProject) && Roddy.isStrictModeEnabled()) {
```

Comment:
The new corrected exit != 0 when there is a validation error in the project XML.